odata-json-format/19 Batch Requests and Responses.md

```diff
@@ -101,8 +101,7 @@ atomicity group, all requests in that group fail with
 The value of `headers` is an object whose name/value pairs
 represent request headers. The name of each pair MUST be the lower-case
 header name; the value is a string containing the header-encoded value
-of the header. The `headers` object MUST contain a name/value
-pair with the name `content-type` whose value is the media type.
```

Comment:
This should be restored (at least as guidance to the client) and co-located with the new paragraph describing the behavior of services that allow omitting the content-type.